cli.js

```diff
@@ -3,13 +3,18 @@
 var meow = require('meow');
 var supportsColor = require('supports-color');
 
-meow({
+var cli = meow({
 	help: [
 		'Example',
 		'  $ supports-color',
+		'  $ supports-color --256',
+		'  $ supports-color --16m',
 		'',
```

Comment:
We should probably explain the flags in an `Options` section. Like: https://github.com/sindresorhus/fkill-cli#usage

The readme also needs to be synced with the new `--help` output.